add catalog task solution

PodvAx opened the pull request:
- [DEMO LINK](https://PodvAx.github.io/layout_catalog/)
- [TEST REPORT LINK](https://PodvAx.github.io/layout_catalog/report/html_report/)

denys-danyliuk commented on src/styles/blocks/card.scss:
```diff
@@ -0,0 +1,117 @@
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-top: 40px;
+    margin-bottom: 4px;
```
5'th point in checklist

denys-danyliuk commented on src/styles/blocks/main-content.scss:
```diff
@@ -0,0 +1,35 @@
+  place-items: center;
+  place-content: center;
+  grid-template-columns: columnsInLayout(1);
+  // grid-template-columns: repeat(4, #{$card-width});
```
delete comments

denys-danyliuk commented on src/styles/blocks/card.scss:
```diff
@@ -0,0 +1,117 @@
+
+    &:hover {
+      transform: scale(1.2);
+      border: 3px solid $grayElements;
```
don't add new border on hover, instead add one with transparent border, and it's color on hover, also mention it in transition

mykhalenych commented on src/styles/blocks/card.scss:
```diff
@@ -0,0 +1,120 @@
+  border-radius: 5px;
+
+  &__image {
+    // display: block;
```
delete all comments

mykhalenych commented on src/styles/blocks/card.scss:
```diff
@@ -0,0 +1,120 @@
+.card {
+  box-sizing: border-box;
```
move to `*` selector and delete in all other places

mykhalenych commented on src/styles/blocks/card.scss:
```diff
@@ -0,0 +1,120 @@
+  }
+
+  &__info-text {
+    font-weight: 400;
```
it is by default

mykhalenych commented on src/index.html:
```diff
@@ -1,12 +1,442 @@
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__info-text card__info-text--product-code">
```
why do you use 3 classes? I think you can you only one

PodvAx replied:
Of course i can. But can i use element with modifier without element itself according to the BEM methodology? And there are only 2 classes.

PodvAx commented:
Where can be a problem? When I add a border of 3px to the picture, I understand that I should change width, height, paddings and margins, so the indents between elements can be perfect. But when I removed 3px from margin-top of card__title, tests did not passed, when i removed 6px it didn't passed, too. But when i removed 7px it passed. So what i do wrong?

Esceype reviewed (approved):
GJ!!!
About your question, you can remove border, make margin-top for card__title like in mockup and provide for img inline-paddings

PodvAx commented:
> GJ!!! About your question, you can remove border, make margin-top for card__title like in mockup and provide for img inline-paddings

Thanks! But I already had the image without border and added it on hover. But @denys-danyliuk said I don't need this, and i should add transparent border. So that's why I had this question.
